src/main/java/graphql/analysis/MaxQueryComplexityInstrumentation.java

```diff
@@ -21,7 +21,7 @@
 /**
  * Prevents execution if the query complexity is greater than the specified maxComplexity.
  *
- * Use the {@link Function<QueryComplexityInfo, Boolean>} parameter to supply a function to perform a custom action when the max complexity
+ * Use the `Function&lt;QueryComplexityInfo Boolean&gt;` parameter to supply a function to perform a custom action when the max complexity
  * is exceeded. If the function returns {@code true} a {@link AbortExecutionException} is thrown.
  */
```

Comment:
This was stopping the command line version from building ebacuse of JavaDoc errors.  Clearly our builds dont run the same profile of JavaDoc as I do.  Not sure why